db/schema.rb

```diff
@@ -15,6 +15,7 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
+  enable_extension "pgcrypto"
```

Comment:
al, can you run a local db:reseed/reload as this extension should have been removed